packages/scaffold/src/client.ts

```diff
@@ -177,6 +178,10 @@ export class Web3ModalScaffold {
       AccountController.setAddressExplorerUrl(addressExplorerUrl)
     }
 
+  protected setMetadata(metadata: OptionsControllerState['metadata']) {
```

Comment:
This function is still here, but it is not needed, correct? You set metadata in `initControllers`